ecl/hql/hqlexpr.cpp

```diff
@@ -7581,8 +7581,8 @@ IHqlExpression * createJavadocAnnotation(IHqlExpression * _ownedBody, IPropertyT
 
 //==============================================================================================================
 
-CFileContents::CFileContents(IFile * _file, ISourcePath * _sourcePath, bool _isSigned)
-  : file(_file), sourcePath(_sourcePath), implicitlySigned(_isSigned)
+CFileContents::CFileContents(IFile * _file, ISourcePath * _sourcePath, bool _isSigned, IHqlExpression * _gpgSignature)
```

Comment:
minor: I'm slightly wondering about the use of an IHqlExpression in this class.  This is a more general purpose class (it could possibly move into jlib) and containing an IHqlExpression doesn't seem quite right.  However there are some benefits and it is unlikely to cause issues, so I wouldn't change it.